graph_altimeter/scan/__init__.py

```diff
@@ -155,3 +161,26 @@ def remove_scan_files(temp_dir, scan_id):
         shutil.rmtree(scan_path)
     except FileNotFoundError:
         logger.debug("temp directory %s not found", scan_path)
+
+
+class InvalidAWSAccount(Exception):
+    """Returned when normalized an AWS account identifier if the format is not
+    recognized."""
+
+    def __init__(self, AWS_account):
+        super().__init__(
+            f'the format of the AWS_account: {AWS_account} is invalid'
+        )
+
+
+def normalize_account_id(aws_account):
```

Comment:
s/normalize_account_id/normalize_aws_account/ ?

Otherwise, it is hard to know that this refers to AWS from the package perspective.

If you change it, remember to modify test names and comments in tests/test_normalize_account_id.py